agent/Dockerfile

```
@@ -0,0 +1,38 @@
+FROM oven/bun:1 AS base
+
+# Pin versions for reproducible builds
+ARG DOCKER_CLI_VERSION=24.0.9
+ARG DOCKER_COMPOSE_VERSION=2.32.4
+
+# Install Docker CLI (static binary — no daemon) and Docker Compose v2 plugin
+RUN apt-get update && \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)